src/main/java/com/onevizion/scmdb/SqlScriptExecutor.java

```diff
@@ -58,12 +59,16 @@ public class SqlScriptExecutor {
     private DataSource pkgDataSource;
 
     public int execute(SqlScript script) {
+        return execute(script, false);
+    }
+
+    private int execute(SqlScript script, boolean isCompileSchemas) {
```

Comment:
Propose to refactor this method to `private int execute(SqlScript script, File wrapperScriptFile)` and update `executeResourceScript` to always provide `"script_wrapper_fail_on_error.sql"`. Also remove `isSkipInvalidsCompilation` from `getTmpWrapperScript()`
![image](https://user-images.githubusercontent.com/3282060/219327159-7934677c-e24e-47b6-b939-b82561ffecef.png)
![image](https://user-images.githubusercontent.com/3282060/219327267-a1fd2d14-c0f6-4eb7-bad1-763cff65eb30.png)

Reply:
Done